livefs_edit/actions.py

```diff
@@ -70,16 +70,37 @@ class LayerfsLoc(enum.Enum):
     INITRD = enum.auto()
 
 
+def _find_file_in_initrd(ctxt, relpath: str) -> Optional[str]:
+    dir = pathlib.Path(unpack_initrd(ctxt))
+    potential_file = dir / relpath
+    if potential_file.exists():
+        return str(potential_file)
+
+    for subdir in dir.iterdir():
+        if not subdir.is_dir():
```

Comment:
So of course I look at this and wonder about ordering. I think technically the last entry wins (sorted by order in the concatenation of cpio archives, which translates to sorting by name in the directory) but the chance of this mattering seems low.

(It also  seems very likely that any path we actually care about will in practice be in the final archive).

I don't know what to do with these thoughts. Maybe just suppress them?

Reply:
If I understand the scenario correctly, we're talking about the case where there are multiple copies of the file name we're looking for in question, and we want the one in the latest directory in the list?  If so, I think that's a bad initrd in general, but I think we can handle that by reversing the order of iteration, searching the last cpio dir first and moving to earlier ones if needed.  Is that what you mean?

Reply:
> If so, I think that's a bad initrd in general

Well this is why I wonder if we do anything. 

> I think we can handle that by reversing the order of iteration, searching the last cpio dir first and moving to earlier ones if needed. Is that what you mean?

Yes, that sounds like the right thing I guess. sorted(path.iterdir(), reverse=True) or something I guess?

(BTW, moving livefs-editor to use pathlib.Paths everywhere would be a good thing I think)